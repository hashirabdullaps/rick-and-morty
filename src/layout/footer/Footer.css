.footer {
    display: flex;
    justify-content: center;
    align-items: center;
    background-color: #e9e9e9 !important;
    width: 100%;
  }
  
  .footer__link {
    text-decoration: none;
    letter-spacing: 3px;
  }
  