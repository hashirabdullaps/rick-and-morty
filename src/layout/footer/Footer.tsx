import { Layout } from "antd";

import './Footer.css';

const Footer = () => {
    return <Layout.Footer className="footer">
        <a className="footer__link" href='/'>
            Rick and Morty by Hashir Â© 2022
        </a>
    </Layout.Footer>
}

export default Footer;