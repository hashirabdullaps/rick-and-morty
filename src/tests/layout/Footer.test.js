import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../../layout/footer/Footer';

test('should test Footer component', () => {
    const component = renderer.create(<Footer />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});