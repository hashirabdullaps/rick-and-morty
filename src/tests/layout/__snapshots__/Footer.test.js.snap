// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Footer component 1`] = `
<footer
  className="ant-layout-footer footer"
>
  <a
    className="footer__link"
    href="/"
  >
    Rick and Morty by Hashir Â© 2022
  </a>
</footer>
`;
